src/screens/Welcome.js

```diff
@@ -43,14 +43,14 @@ class Welcome extends React.Component {
               <img className="hathor-logo" src={logo} alt="" />
               <p className="mt-4 mb-4">Welome to Hathor Wallet!</p>
             </div>
-            <p className="mb-4">This wallet is connected to a <strong>testnet</strong>.</p>
-            <p>Your HTR and other tokens may be reset at any time.</p>
-            <p>If one offers to sell some tokens to you, one is a scammer.</p>
+            <p className="mb-4">This wallet is connected to the <strong>mainnet</strong>.</p>
+            <p>In this wallet you can send tokens and see the history of your transactions. Besides that, you can also create and manage new tokens.</p>
+            <p>You are responsible for your tokens and the safety of them.</p>
```

Comment:
Suggestion: You should never share your seed with other people. Anyone with access to your seed have full control of your balance.

Reply:
At this point we didn't reach the seed part. Maybe talking about it here would be a bit confusing.

Reply:
I agree, that's one of the reasons I haven't mentioned the words here. Maybe even talk about the "safety of tokens" is confusing but I feel it's good so we can have the link to the website if someone wants to read more.

Reply:
You're right. Maybe: "You should never share nothing besides your addresses to other people. You're fully responsible for keeping your tokens safe."

What do you think?

Reply:
I'd just tweat: "You should never share **_any information_** besides"

Reply:
Great! Thanks!